### PR TITLE
OSDOCS-5275-link-redo:Updated vSphere release notes by adding links t…

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.13

Issue:
[OSDOCS-5725](https://issues.redhat.com/browse/OSDOCS-5725)

Link to docs preview:
* [Installation and update](https://59464--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-installation-and-update)
* [Multiple regions and zones configuration for a cluster on vSphere](https://59464--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-post-installation)

QE review:
- [X] QE has approved this change. Not needed for adding internal doc links to release notes.

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -203,6 +207,8 @@ For more information about scaling a user-provisioned cluster by using the BMO,
 ==== Specifying multiple failure domains for your cluster on VSphere
 As an administrator, you can specify multiple failure domains for your {product-title} cluster that runs on a VMware VSphere instance. This means that you can distribute key control planes and workload elements among varied hardware resources for a datacenter. Additionally, you can configure your cluster to use a multiple layer 2 network configuration, so that data transfer among nodes can span across multiple networks.
 
+For more information, see xref:../post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc#post-install-vsphere-zones-regions-configuration_post-install-vsphere-zones-regions-configuration[Specifying multiple failure domains for your cluster on VSphere].
```
Comment (opayne1) [inline release_notes/ocp-4-13-release-notes.adoc:210]: This link takes me to "Multiple regions and zones configuration for a cluster on vSphere" not "Specifying multiple failure domains for your cluster on VSphere" Maybe the link text needs changing?

**Review by opayne1 — COMMENTED** [review]
Just one small comment otherwise lgtm

File: release_notes/ocp-4-13-release-notes.adoc (hunk 1 above)
Comment (dfitzmau) [reply release_notes/ocp-4-13-release-notes.adoc]: Thanks for spotting this. I updated the link to now point to the specific module.

**Review by kelbrown20 — COMMENTED** [review]
Updates LGTM as well! Great work! 
Merging

**Pull request merged**